.github/workflows/ci_integration.yml

```diff
@@ -282,92 +282,72 @@ jobs:
 
   container-build:
     name: "Build Container Images (GHCR Only)"
-    runs-on: "ubuntu-20.04"
+    runs-on: "ubuntu-22.04"
     if: "github.event_name == 'push' && (github.ref == 'refs/heads/develop' || github.ref == 'refs/heads/next')"
     needs:
       - "integration-test"
-    strategy:
-      fail-fast: true
-      matrix:
-        python-version: [ "3.7", "3.8", "3.9", "3.10" ]
-    env:
-      INVOKE_NAUTOBOT_PYTHON_VER: "${{ matrix.python-version }}"
     steps:
+      - name: "Configuration"
+        id: "config"
+        shell: "bash"
+        run: |
+          ref="${{ github.ref }}"
+          branch="${ref#refs/heads/}"
+
+          case "$branch" in
+            "develop")
+              min_python_version="3.7"
+              max_python_version="3.10"
```

Comment:
We'll want to be sure to change this to 3.8,3.11 as soon as `develop-1.6` collapses down into `develop` and brings in #4118.

Reply:
I say we let `develop-1.6` merge down before merging this so we can add the suggestion now.

Reply:
Updated to use 3.8 and 3.11 for develop as well